README.md

```diff
@@ -90,11 +94,6 @@ environmental variable `SGFROOT` to point to this location.  That is, in bash,
    - either ```export PATH=$PATH:/path/to/sgf/bin```,
    - or ```export SGFROOT=/path/to/sgf/bin```
 
-#### Forked DataSynthesizer
```

Comment:
Removed this just because it gets installed as part of the dependencies anyway (`env-configuration/requirements.txt`)